spec/acceptance/agent_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-['2.4', '3.2', '3.4', '4.0'].each do |version|
+['2.4', '3.2', '3.4', '4.0', '4.2', '4.4'].each do |version|
```

Comment:
this only adds them for the agent. Do you also want to do this for the server? (maybe also have a look at https://github.com/voxpupuli/puppet-zabbix/pull/674 )

Reply:
I couldn't find a similar version specific testing for the server resources and I'm currently short on time for restructuring the server_spec.rb File. Sorry! :)